src/satellite/.eslintrc.js

```diff
@@ -1,20 +1,15 @@
 module.exports = {
-  env: {
-    node: true,
-    commonjs: true,
-    es2021: true,
-    jest: true,
-  },
-  extends: 'eslint:recommended',
-  parserOptions: {
-    ecmaVersion: 13,
-  },
-  plugins: ['anti-trojan-source', 'jest'],
```

Comment:
should we keep this plugin: `anti-trojan-source`?

Reply:
we already have it in `@senecacdot/eslint-config-telescope`